### DISCUSSION TITLE
Meta Issues Filter View

### DISCUSSION THREAD

**Discussion by 0x4007** [devpool-directory/devpool-directory#475] [Ideas]
Bounties for this repository should be easy to find with some type of search filter. Unfortunately the GitHub search filters aren't extremely sophisticated, so we would need to filter by a single label or some identifier. 

The direct solution seems to be to add a special label to the meta issues (bounties posted directly on this repository) but it seems inconsistent with how we have set up bounties everywhere else. 

The alternative is to add a special label on automatically added bounties which we could filter out.

Unfortunately we can not filter by "not author" for two different authors (currently we have ubiquibot and rndquu users who imported the network bounties) GitHub only supports one "not"

**Comment by rndquu** [comment]
> currently we have ubiquibot and rndquu users who imported the network bounties

We can recreate the `devpool` repository. This way all issues will be created by the "ubiquibot"

**Reply by 0x4007** [reply]
Can you do that and rename it to `devpool-directory`?

**Comment by 0x4007** [comment]
I also think we should include priority and time so that bounty hunters can easily sort by short time and high priority issues, for example.

**Reply by rndquu** [reply]
Pls check this issue: https://github.com/ubiquity/devpool-directory-bounties/issues/3
